sass/_FacetSearch.scss

```diff
@@ -39,6 +43,12 @@
   .coveo-facet-value-label {
     padding-right: 15px;
   }
+  .CoveoFacet & {
+    margin-top: -1px;
+  }
+  .CoveoCategoryFacet & {
+    line-height: normal;
+  }
```

Comment:
This is to compensate for `FacetSearch`'s results now being inside `CategoryFacet` when its search box is clicked. `CategoryFacet` gives all its descendants a `line-height`.